resources/ts/Components/Integrations/Detail/OrganizersInfo.tsx

```diff
@@ -77,6 +77,7 @@ const OrganizersSection = ({
               {sectionName === "Live" && (
                 <div>
                   <ButtonIcon
+                    id={organizer.name[i18n.language]}
```

Comment:
Maybe add a data-testid instead? Since that is what you can use in playwright by getByTestId